xbmc/Application.cpp

```diff
@@ -4611,7 +4611,7 @@ void CApplication::ProcessSlow()
 
   // Temporarely pause pausable jobs when viewing video/picture
   int currentWindow = g_windowManager.GetActiveWindow();
-  if (CurrentFileItem().IsVideo() || CurrentFileItem().IsPicture() || currentWindow == WINDOW_FULLSCREEN_VIDEO || currentWindow == WINDOW_SLIDESHOW)
+  if (currentWindow == WINDOW_FULLSCREEN_VIDEO || currentWindow == WINDOW_SLIDESHOW)
```

Comment:
@FernetMenta: but it basically cripples any library views because you don't get any detailed information or artwork.